bedrock/firefox/templates/firefox/features/sync.html

```diff
@@ -0,0 +1,165 @@
+      <div id="sync-intro-copy">
+        {# Warning Text #}
+        <div class="show-fx-29-30 show-fx-28-older warning">
+          <p>
+            <span>{{ _('It looks like you’re running an older version of Firefox.') }}</span>
+          </p>
+        </div>
+
+        <div class="show-not-fx warning">
+          <p>
+            <span>
+            {% if l10n_has_tag('firefox_sync_non_fx') %}
+              {{ _('Sync is just one of the great features you’ll only get with Firefox.') }}
+            {% else %}
+              {{ _('Sync is just one of the great features you’ll only get with Firefox. Discover them all!') }}
+            {% endif %}
+            </span>
+          </p>
+        </div>
+
+      {% block intro %}
+        <div class="show-fx-31-signed-in">
+          <h1>{{ _('Ready, Set, Sync') }}</h1>
+          <p>
+            {{ _('You’re signed up and ready to access <em>your</em> Firefox anywhere, anytime.') }}
+          </p>
+        </div>
+        <div class="show-fx-31-signed-out show-fx-android show-not-fx show-default">
+          <h1>{{ _('Take your Web with you') }}</h1>
+          <p>
+            {{ _('Sync Firefox wherever you use it to access your bookmarks, passwords, tabs and more from any smartphone, tablet or computer.') }}
+          </p>
+        </div>
+      {% endblock %}
+      </div>{#--/#sync-intro-copy--#}
+
+      <div id="sync-intro-cta">
+        <div class="show-fx-31-signed-in">
+          <ul class="primary-buttons">
+            <li>
+              {{ google_play_button(id='cta-android') }}
+            </li>
+            <li>
+              <a href="{{ firefox_ios_url('mozorg-sync_page-appstore-button') }}" data-link-type="download" data-download-os="iOS" id="cta-ios">
+                <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+              </a>
+            </li>
+          </ul>
+
+          <p class="button-footer">
+            <a href="https://support.mozilla.org/kb/how-do-i-set-up-firefox-sync" class="ga-link" data-interaction="outbound link click">
+              {{ _('Need help?') }}
+            </a>
+          </p>
+        </div>
+
+        <div class="show-fx-31-signed-out show-fx-android">
+          <button class="button button-hollow" data-button-name="Get started with Sync" data-cta-position="Primary" id="cta-sync">
+            {{ _('Get started with Sync') }}
+          </button>
+        </div>
+
+        <div class="show-fx-29-30 show-fx-28-older">
+          <a href="https://support.mozilla.org/kb/update-firefox-latest-version" class="button button-hollow" data-interaction="button click" data-element-action="Sync CTA" id="cta-update">
+            {{ _('Update your Firefox') }}
+          </a>
+        </div>
+
+        <div class="show-not-fx">
+          {{ download_firefox(button_color='button-arrow', download_location='secondary cta') }}
```

Comment:
Is `secondary cta` a value Peter is mapping? It wasn't in the original list of [proposed values](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templatetags/helpers.py#L118-L119) so seems unique to this page atm.